DOCS/license-inclusion-principles.md

```diff
@@ -4,30 +4,37 @@ Determining whether a candidate license should be included on the SPDX License L
-* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
-* The license has identifiable and stable text; it is not in the midst of drafting.
-* The license steward, if any, is committed to not modifying after addition to the list and to versioning new versions in the future.
+<ol type="A">
+ <li>The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.</li>
+ <li>All OSI-approved licenses will be included on the SPDX License List.</li>
+ <li>This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.</li>
+ <li>Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.</li>
+ <li>The license has identifiable and stable text; it is not in the midst of drafting.</li>
+ <li>The license steward, if any, is committed to not modifying after addition to the list and to versioning new versions in the future.</li>
+</ol>
   
 **Other factors (roughly in order of importance):**
+
 1. The license substantially complies with one of the following open source definitions (even if not submitted for approval or these organization have not considered the license):
   * [Open Source Definition (OSD)](https://opensource.org/osd) from the Open Source Initiative (OSI)
   * [Free Software Definition](https://www.gnu.org/philosophy/free-sw.en.html) from the Free Software Foundation (FSF)
   * [Open Source Hardware Definition](https://www.oshwa.org/definition/) from the Open Source Hardware Association (OSHWA)
   * [Open Definition](http://opendefinition.org/od/2.1/en/) from the Open Knowledge Foundation
   * [Free Cultural Works definition](https://freedomdefined.org/Definition) from Freedom Defined.org
-2. The license is structured to be generally usable by anyone. It is not specific to one project, consortium or corporation.
-3. The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstrated via use in many projects, or in one or a few significant projects. For new licenses, there are definitive plans for the license to be used in one or a few significant projects.
+
+2. The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstrated via use in many projects, or in one or a few significant projects. For new licenses, there are definitive plans for the license to be used in one or a few significant projects.
+
+3. The license is structured to be generally usable by anyone. It is not specific to one project, consortium or corporation.
```

Comment:
We haven't agreed yet to swap the order for 2 and 3 -- I think this is to be discussed later in a change proposal, see https://github.com/spdx/license-list-XML/issues/1624#issuecomment-1255244966